staging/src/k8s.io/kms/apis/v1beta1/api.proto

```diff
@@ -19,9 +19,11 @@ syntax = "proto3";
 
 package v1beta1;
 option go_package = "k8s.io/kms/apis/v1beta1";
+option deprecated = true;
 
 // This service defines the public APIs for remote KMS provider.
 service KeyManagementService {
+    option deprecated = true;
```

Comment:
Setting this option adds a `// Deprecated: Do not use.` comment on top of the methods. For the messages, I explicitly added the descriptive comment instead of using this.

Reply:
I'm not sure the "Deprecated: Do not use" comments are that helpful... I think the feature gate, warning, and explanation on the messages is sufficient

Reply:
Thanks, Jordan! removed it.